examples/step-19/doc/intro.dox

```diff
@@ -60,7 +60,7 @@ density. This is augmented by boundary conditions that we will choose as follows
    && \text{on}\; \partial\Omega\setminus\Gamma_\text{cathode}\setminus\Gamma_\text{anode}.
 @f}
 In other words, we prescribe voltages $+V_0$ and $-V_0$ at the two electrodes
-and isolating (Neumann) boundary conditions elsewhere. Since the dynamics of the
+and insulating (Neumann) boundary conditions elsewhere. Since the dynamics of the
```

Comment:
I think it should be *insulating* for electrical comments, but given that my native language does not distinguish between isolate and insulate I let someone else check whether I am right.

Reply:
I must have been speaking too much German that day. You're completely right.